bouncer/blacklist/admin.py

```diff
@@ -4,5 +4,5 @@
 
 @admin.register(models.IPEntry, models.EmailEntry, models.EmailHostEntry)
 class EntryAdmin(admin.ModelAdmin):
-    fields = ["entry_value", "reason"]
-    search_fields = ["entry_value"]
+    fields = ["entry_value", "lower_case_entry_value", "reason"]
```

Comment:
Lower case entry value is not something that makes sense to be visible in the admin, as the admin can do nothing with it. It would be best to use something like [`editable=False`](https://docs.djangoproject.com/en/2.0/ref/models/fields/#editable).